app/javascript/src/common/dummy.js

```diff
@@ -0,0 +1,91 @@
+export const ROLES = ["Owner", "User"];
```

Comment:
Read: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Constants%20file%20can%20be%20used%20to%20store%20initial%20values%2C%20default%20values%2C%20etc.

- These values are literal constants. Thus they don't belong in `common`.
- Are all the constant values defined in this file being used at more than one component across your codebase? If not, then they don't belong in the global scope. Read: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Keep%20the%20files%20in%20closest%20scope

Reply:
Sorry about that. Those initial values used to be in the `constants.js` file until 5e4552f commit. While creating `DashboardContext` some flawed logic made me put it in the common folder.